test/Test.hs

```diff
@@ -208,6 +287,53 @@ jobsTable =
     ]
   }
 
+personTable :: Table
+personTable =
+  tblTable
+  { tblName = "person_test"
+  , tblVersion = 1
+  , tblColumns =
+    [ tblColumn { colName = "id",            colType = BigSerialT
+                , colNullable = False }
+    , tblColumn { colName = "name",          colType = TextT
+                , colNullable = False }
+    , tblColumn { colName = "age", colType = IntegerT
+                , colNullable = False }
+    ]
+  , tblPrimaryKey = pkOnColumn "id"
+  }
+
+duplicatingJobsTable :: Table
+duplicatingJobsTable =
+  tblTable
+  { tblName = "consumers_test_duplicating_jobs"
+  , tblVersion = 1
+  , tblColumns =
+    [ tblColumn { colName = "id",          colType = BigSerialT
+                , colNullable = False }
+    , tblColumn { colName = "run_at",      colType = TimestampWithZoneT
+                , colNullable = True }
+    , tblColumn { colName = "finished_at", colType = TimestampWithZoneT
+                , colNullable = True }
+    , tblColumn { colName = "reserved_by", colType = BigIntT
+                , colNullable = True }
+    , tblColumn { colName = "attempts",    colType = IntegerT
+                , colNullable = False }
+
+      -- Non-obligatory field "countdown". Really more of a count
+      -- and not a countdown, but name is kept to that we can reuse
+      -- `putJob` function.
+    , tblColumn { colName = "countdown",    colType = IntegerT
+                , colNullable = False }
```

Comment:
There's quite a lot of whitespace before `colType`. While I kind of get what the formatting means, it's confusing and I think it would be better to simply write it on the next line.